### PR TITLE
Altered smart inventory details

### PR DESCRIPTION
**Pull request by ianf77**
Removed information about changing variables

Clarify documentation for editing host/group variables in Smart Inventories

https://issues.redhat.com/browse/AAP-10694

### CODE REVIEW

File: downstream/modules/platform/ref-controller-smart-inventories.adoc [hunk 1]
```diff
@@ -51,14 +51,12 @@ Organization administrators have implicit "admin" access to all inventories insi
 Administrators of the Smart Inventory can grant other users (who are not also admins of your organization) permissions such as "use" and "adhoc" to the smart inventory. 
 These permit the actions indicated by the role, as with other standard inventories. 
 However, this does not grant any special permissions to hosts (which live in a different inventory).
-It does not permit direct read permission to hosts, or permit them to see additional hosts under `/#/hosts/`, although they can still view
-the hosts under the smart inventory host list.
+It does not permit direct read permission to hosts, or permit them to see additional hosts under `/#/hosts/`, although they can still view the hosts under the smart inventory host list.
 
 In some situations, you can modify the following:
```
Comment (tvo318) [inline downstream/modules/platform/ref-controller-smart-inventories.adoc:56]: Isn't this the initial issue is that you can't modify?

Comment (tvo318) [reply downstream/modules/platform/ref-controller-smart-inventories.adoc:56]: Nevermind, you can still modify those, just the last one Variables on Host and Group aren't changeable. My bad.

**Review by tvo318 — APPROVED** [review]
Looks good.

**Review by EMcWhinn — APPROVED** [review]
LGTM

**Review by mdinwoodie — COMMENTED** [review]
These changes look good to me.

**Pull request merged**